atomicapp/applogging.py

```diff
@@ -121,7 +121,7 @@ def setup_logging(verbose=None, quiet=None, logtype=None):
         if logging_level == logging.DEBUG:
             formatstr = '%(asctime)s - [%(levelname)s] - %(longerfilename)s - %(message)s'
         else:
-            formatstr = '%(asctime)s - [%(levelname)s] - %(filename)s - %(message)s'
+            formatstr = '[%(levelname)s] - %(filename)s - %(message)s'
 
```

Comment:
Unfortunately I wasn't able to get this to work :( with your customOutputFormatter function. I did however add a comment however above where I specify %s.